taxonomy: Add Canadian front-of-package nutrition symbol (#11284)
openfoodfacts/openfoodfacts-server: taxonomies/labels.txt

image:fr: prix-juste-producteur.90x90.png
web:fr: https://prixjuste.be/

########################## Canadian front-of-package nutrition symbol ##########################

# There are 56 possible variations. 7 different combinations for the ingredients, 4 different language versions and 2 orientations (vertical/horizontal).

> 

Reviewer: ```suggestion
# We have only kept the horizontal versions of the images.
```

en: Canadian Front-of-package nutrition symbol
fr: Symbole nutritionnel sur le devant de l'emballage
country:en: en:Canada

> description:en: We're improving food labels. A front-of-package nutrition symbol is required on foods that are high in one or more of these nutrients: sodium, sugars, saturated fat. The food industry has been given until January 1, 2026 to make this change. However, you may start seeing the front-of-package nutrition symbol earlier.

Reviewer: ```suggestion
description:en: In Canada, a front-of-package nutrition symbol is required on foods that are high in one or more of these nutrients: sodium, sugars, saturated fat. The food industry has been given until January 1, 2026 to make this change. However, you may start seeing the front-of-package nutrition symbol earlier.
description:fr: Au Canada, un symbole nutritionnel sur le devant de l'emballage doit être apposé sur les aliments riches en un ou plusieurs de ces nutriments : sodium, sucres, gras saturés. L'industrie alimentaire a jusqu'au 1er janvier 2026 pour procéder à ce changement. Toutefois, il est possible que vous commenciez à voir le symbole nutritionnel sur la face avant des emballages plus tôt.
```

label_categories:en: nutritional-values
web:en: https://www.canada.ca/en/health-canada/services/food-nutrition/nutrition-labelling/front-package.html
web:fr: https://www.canada.ca/fr/sante-canada/services/aliments-nutrition/etiquetage-nutritionnel/devant-emballage.html

< en:Canadian front-of-package nutrition symbol

> en: High in Sat fat Sugars Sodium (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sat fat, Sugars and Sodium (Health Canada)
```

Reviewer: This is what the user will read on the product page. There is no need to add "horizontal", it could be confusing for him/her. And I have added punctuation and a final "and" to make it better to read.

> fr: Élevé en Gras sat. Sucres Sodium (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Gras sat., Sucres and Sodium (Santé Canada)
```

image:xx: high-in-saturated-fat-sugars-sodium-health-canada-eleve-en-gras-satures-sucres-sodium-sante-canada.171x90.svg
image:en: high-in-saturated-fat-sugars-sodium-health-canada.120x90.svg
image:fr: eleve-en-gras-satures-sucres-sodium-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sat fat Sugars (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sat fat and Sugars (Health Canada)
```

> fr: Élevé en Gras sat. Sucres (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Gras sat. et Sucres (Santé Canada)
```

image:xx: high-in-saturated-fat-sugars-health-canada-eleve-en-gras-satures-sucres-sante-canada.171x90.svg
image:en: high-in-saturated-fat-sugars-health-canada.120x90.svg
image:fr: eleve-en-gras-satures-sucres-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sugars Sodium (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sugars and Sodium (Health Canada)
```

> fr: Élevé en Sucres Sodium (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Sucres et Sodium (Santé Canada)
```

image:xx: high-in-sugars-sodium-health-canada-eleve-en-sucres-sodium-sante-canada.171x90.svg
image:en: high-in-sugars-sodium-health-canada.120x90.svg
image:fr: eleve-en-sucres-sodium-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sat fat Sodium (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sat fat and Sodium (Health Canada)
```

> fr: Élevé en Gras sat. Sodium (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Gras sat. et Sodium (Santé Canada)
```

image:xx: high-in-saturated-fat-sodium-health-canada-eleve-en-gras-satures-sodium-sante-canada.171x90.svg
image:en: high-in-saturated-fat-sodium-health-canada.120x90.svg
image:fr: eleve-en-gras-satures-sodium-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sat fat (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sat fat (Health Canada)
```

> fr: Élevé en Gras sat. (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Gras sat. (Santé Canada)
```

image:xx: high-in-saturated-fat-health-canada-eleve-en-gras-satures-sante-canada.171x90.svg
image:en: high-in-saturated-fat-health-canada.120x90.svg
image:fr: eleve-en-gras-satures-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sugars (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sugars (Health Canada)
```

> fr: Élevé en Sucres (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Sucres (Santé Canada)
```

image:xx: high-in-sugars-health-canada-eleve-en-sucres-sante-canada.171x90.svg
image:en: high-in-sugars-health-canada.120x90.svg
image:fr: eleve-en-sucres-sante-canada.120x90.svg

< en:Canadian front-of-package nutrition symbol

> en: High in Sodium (Health Canada) - Horizontal

Reviewer: ```suggestion
en: High in Sodium (Health Canada)
```

> fr: Élevé en Sodium (Santé Canada) - Horizontale

Reviewer: ```suggestion
fr: Élevé en Sodium (Santé Canada)
```

en: Canadian Front-of-package nutrition symbol
fr: Symbole nutritionnel sur le devant de l'emballage
country:en: en:Canada
description:en: In Canada, a front-of-package nutrition symbol is required on foods that are high in one or more of these nutrients: sodium, sugars, saturated fat. The food industry has been given until January 1, 2026 to make this change. However, you may start seeing the front-of-package nutrition symbol earlier.
description:fr: Au Canada, un symbole nutritionnel sur le devant de l'emballage doit être apposé sur les aliments riches en un ou plusieurs de ces nutriments : sodium, sucres, gras saturés. L'industrie alimentaire a jusqu'au 1er janvier 2026 pour procéder à ce changement. Toutefois, il est possible que vous commenciez à voir le symbole nutritionnel sur la face avant des emballages plus tôt.
label_categories:en: nutritional-values
web:en: https://www.canada.ca/en/health-canada/services/food-nutrition/nutrition-labelling/front-package.html
web:fr: https://www.canada.ca/fr/sante-canada/services/aliments-nutrition/etiquetage-nutritionnel/devant-emballage.html

< en:Canadian front-of-package nutrition symbol

> en: High in Sat fat, Sugars and Sodium (Health Canada)

Reviewer: Sorry, my bad :(
